tests/integration/messages_with_implict_size/generated/rflx-universal-message.ads

```diff
@@ -440,12 +442,8 @@ is
        and Valid (Ctx, F_Message_Type)
        and Get_Message_Type (Ctx) = Val
        and (if Structural_Valid_Message (Ctx) then Message_Last (Ctx) = Field_Last (Ctx, F_Message_Type))
-       and Invalid (Ctx, F_Length)
-       and Invalid (Ctx, F_Data)
-       and Invalid (Ctx, F_Option_Types)
-       and Invalid (Ctx, F_Options)
-       and Invalid (Ctx, F_Value)
-       and Invalid (Ctx, F_Values)
+       and (for all F in Field =>
+               (if F > F_Message_Type then Invalid (Ctx, F)))
```

Comment:
Could the following variant be an even better one? It removes the if expression.

```ada
       and (for all F in F_Length .. F_Values => Invalid (Ctx, F))
```

Reply:
Yes that looks much better. I will do that.

Reply:
done